django/test/runner.py

```diff
@@ -581,6 +594,10 @@ def add_arguments(cls, parser):
             ),
         )
 
+    def log(self, msg, level=DEBUG):
```

Comment:
The default level should be `logging.INFO` rather than `DEBUG`. However, I would implement this by making the default `None` (`level=None`) and interpreting `None` as `logging.INFO` in the body of the method.

Reply:
Sure. Just out of curiosity, I wanted to ask, why would you implement it by defining `level = None` as the default value and the setting it to logging.INFO if level = None. Any specific reason or just a convention for better readability?

Reply:
It simplifies getting the default behavior for callers because they can just pass `None`, instead of having to hard-code the default in a second location or invoke the function in a different way (without passing the argument).